cypress/fixtures/constants.ts

```diff
@@ -213,7 +213,14 @@ export class Constants {
         sharedContextApp1H2: 'App 1',
         sharedContextApp2H2: 'App 2',
         sharedContextApp1Paragraph: 'Welcome, Billy',
-        sharedContextApp2Paragraph: 'Welcome, Susan',
+        sharedContextApp2Paragraph: 'Welcome, Susan',,
+        sharedRoutes2App1Header: 'App 1',
```

Comment:
we already have App1 - App2 constants, please use them

Reply:
fixed